### PR TITLE
Update to Dalamud API X, FFXIV ClientStruct changes

### PR DESCRIPTION
**Pull request by redstrate**
This is some of the easy stuff that was changed upstream in Dalamud & CS, to get it closer to building & working. Some of the actor things are intentionally left out because I'm less knowledgeable on that part, that might be a separate PR.

### CODE REVIEW

File: Actors/ActorResolver.cs [hunk 1]
```diff
@@ -1,191 +1,192 @@
-using Dalamud.Plugin.Services;
```
Comment (redstrate) [inline Actors/ActorResolver.cs:1]: Meh, I have no idea why this whole file is marked as changed but it's not for me locally.

Comment (redstrate) [reply Actors/ActorResolver.cs:1]: It's those stupid line endings, lemme see if I I can fix that

Comment (Ottermandias) [reply Actors/ActorResolver.cs:1]: I have done all of this already anyway.

Comment (redstrate) [reply Actors/ActorResolver.cs:1]: Oh, nice! Was a good learning exp anyway

Comment (Ottermandias) [reply Actors/ActorResolver.cs:1]: Sorry if this came off as rude. I still appreciate you trying to help out!

Comment (redstrate) [reply Actors/ActorResolver.cs:1]: lol, don't worry it didn't come across that way at all :D i appreciate the comment though

**Pull request closed without merging**